packages/helm-charts/blockscout/templates/grafana-hook.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
+  annotations:
+    "helm.sh/hook": post-upgrade
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Release.Name }}-rbac
+      initContainers:
+{{ include "celo.blockscout.initContainer.secrets-init" . | indent 6 }}
+      containers:
+      - name: post-install-grafana
+        image: curlimages/curl
+        imagePullPolicy: IfNotPresent
+        command:
+        - /tmp/pod/secrets-init
+        args:
+        - --provider
+        - google
+        - /bin/sh
+        - -c
+        - /opt/scripts/grafana-hook.sh
+        env:
+        - name: GRAFANA_API_TOKEN
+          value: {{ .Values.blockscout.secrets.grafanaCloud | quote }}    
+        - name: GRAFANA_API_ENDPOINT
+          value: {{ .Values.blockscout.grafana_url | quote }} 
```

Same, please stick to the camelCase notation in values names for consistency (existing parameters with _ will be updated in the other PR).
```suggestion
          value: {{ .Values.blockscout.grafanaUrl | quote }} 
```